### PR TITLE
Include required libraries in pkg-config output (SDL2)

### PR DESCRIPTION
**Pull request by ccawley2011**

### CODE REVIEW

**Comment by orgads** [comment]
Can this be merged?

**Comment by slouken** [comment]
> Can this be merged?

Yep!

**Comment by orgads** [comment]
@ccawley2011 can you please rebase it?

**Comment by ccawley2011** [comment]
> @ccawley2011 can you please rebase it?

OK, done.

File: configure [hunk 1]
```diff
@@ -16189,6 +16193,25 @@ fi
...
+    ENABLE_STATIC_FALSE="#"
+else
+    ENABLE_STATIC_TRUE="#"
+    ENABLE_STATIC_FALSE=
+fi
+
+
+
+
+
```
Comment (orgads) [inline configure:16214]: Can you remove all these blank lines?

Comment (ccawley2011) [reply configure:16214]: This file is generated by autotools, so I don't think there's much that can be done about that.

Comment (orgads) [reply configure:16214]: Ah, ok. Thanks for clarifying.

**Comment by orgads** [comment]
@sezero Is a new release planned anytime soon?

**Comment by sezero** [comment]
> @sezero Is a new release planned anytime soon?

Don't know - @slouken and/or @icculus decide that.

**Comment by orgads** [comment]
Thank you! :)

**Comment by orgads** [comment]
Ahh there was a release, but it doesn't include this commit! :/

So when is the *next* release planned @slouken / @icculus?

**Pull request merged**